src/components/InputV2/Address.tsx

```diff
@@ -34,6 +43,28 @@ export const AddressInput = ({
     [error, value]
   )
 
+  // Convert to delegated if eth
+  const delegatedFromEth = useMemo(
+    () => (!error && isEthAddress(value) ? delegatedFromEthAddress(value) : ''),
+    [error, value]
+  )
+
+  // Convert to eth if delegated
+  const ethFromDelegated = useMemo(
+    () =>
+      !error && isDelegatedAddress(value) ? ethAddressFromDelegated(value) : '',
+    [error, value]
+  )
+
+  // Communicate delegated address to parent component
+  useEffect(
+    () => setDelegated(delegatedFromEth),
+    [setDelegated, delegatedFromEth]
+  )
+
+  // Store any converted value for display
+  const converted = delegatedFromEth || ethFromDelegated
```

Comment:
Here i think we should change the text based on which way we're going:

let converted = ''

if (delegatedFromEth) converted = "Converts into ${delegatedFromEth}"
else if (ethFromDelegated) converted = "Represented by ${ethFromDelegated}"

I'm not sure the difficulty on this, but ideally, `delegatedFromEth` and `ethFromDelegated` would be `SmartLink` to the explorer

Reply:
Done!

Reply:
Ooh wait I didn't do the AddressLink

Reply:
Done!